src/main/java/pl/piekoszek/gorskimatches/equation/EquationController.java

```diff
@@ -28,6 +24,6 @@ String fetchString() {
 
     @PostMapping("solution")
     boolean checkAnswer(@Valid @RequestBody Equation equation) {
-        return solvableEquations.checkForCorrectAnswer(equation.quiz, stringValidator.removeSpaces(equation.answer));
+        return solvableEquations.checkForCorrectAnswer(equation.quiz, equation.answer);
```

Comment:
you added spaces to pattern, but removed `removeSpaces`. Now every answer with space inside is `wrong`